### DISCUSSION TITLE
Lazy-loading images and anchor navigation

### DISCUSSION THREAD

**Discussion by rafutek** [facebook/docusaurus#7840] [Q&A]
Hello,

I do not known if it's a bug or not, so I will ask first. When I load a page from a link containing an anchor, I expect to arrive at the anchor level. However, it is not the case, and I think it is because of the images lazy-loading. Indeed, once the page is fully loaded I can navigate beetween sections without problem. Is it a bug or expected behaviour ?

Thank you

**Comment by Josh-Cena** [comment]
It has been a recurring issue. Are you using the latest version?

See https://github.com/facebook/docusaurus/issues/6685

**Reply by rafutek** [reply]
Yes, 2.0.0-rc.1.
